### ISSUE TITLE
`src/models/FieldLayout::getTabs()` method does not set internal _fields array breaking `getField` method

### ISSUE THREAD

**Issue by leevigraham** [verbb/formie#441]
**Description**

`src/models/FieldLayout` extends `CraftFieldLayout` and overrides the `getTabs` method. In `CraftFieldLayout` the `getTabs` method sets an internal `_fields` property

![image](https://user-images.githubusercontent.com/25124/115631215-b97ecd80-a348-11eb-9bfb-af044bb869f8.png)

This is then used in the `getField()` method which I'm using to get the field by it's handle:

![image](https://user-images.githubusercontent.com/25124/115631268-d74c3280-a348-11eb-8c41-fce9335229d8.png)

`src/models/FieldLayout::getFields()` does not set the internal `_fields` property so the `getField` method throws an exception.

- Plugin version: 1.4.0
- Craft version: 3.6.12

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.2](https://github.com/verbb/formie/releases/tag/1.4.2)
